### PR TITLE
Finished putting together the main scene for Alpha

### PR DESCRIPTION
**Pull request by jake-pauls**
- Merged work from all development scenes into `Main`
- Various bug fixes for the environment in the `Main` scene
- Various environment changes/fixes to make abilities and interactions more transparent in the environment
- Fool-proofed both the earth and water puzzles
- Hooked up the main menu to build output

### CODE REVIEW

File: Assets/Scripts/Interaction/BallPickup.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BallPickup : MonoBehaviour, IInteractable
+{
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/BallPickup.cs]: Should be inline with line 5

File: Assets/Scripts/Interaction/Interactable.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
+
+public interface IInteractable
+{
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/Interactable.cs]: Should be inline with line 3

File: Assets/Scripts/Interaction/OrbPickup.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class OrbPickup : MonoBehaviour, IInteractable
+{
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/OrbPickup.cs]: Inline with line 5

File: Assets/Scripts/Interaction/OrbPickup.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
+using System.Collections;
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/OrbPickup.cs]: unused using statement

File: Assets/Scripts/Interaction/BallPickup.cs [hunk 5]
```diff
@@ -0,0 +1,54 @@
+using System.Collections;
+using System.Collections.Generic;
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/BallPickup.cs]: Unused using statements

File: Assets/Scripts/Interaction/PlayerInteraction.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using System.Collections.Generic;
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/PlayerInteraction.cs]: Unused using statements

File: Assets/Scripts/Interaction/PlayerInteraction.cs [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PlayerInteraction : MonoBehaviour
+{
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/PlayerInteraction.cs]: Inline with line 6

File: Assets/Scripts/Interaction/PressurePlate.cs [hunk 8]
```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/PressurePlate.cs]: Unused using statements

File: Assets/Scripts/Interaction/PressurePlate.cs [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+using UnityEngine;
+
+public class PressurePlate : MonoBehaviour
+{
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/PressurePlate.cs]: Bracket should be inline, and not start on a new line

File: Assets/Scripts/Interaction/Tree.cs [hunk 10]
```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
```
Comment (mbajaman) [inline Assets/Scripts/Interaction/Tree.cs]: Line 2 - unused using statement

File: Assets/Scripts/Puzzle/SoundPuzzle.cs [hunk 11]
```diff
@@ -1,3 +1,4 @@
+using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
```
Comment (mbajaman) [inline Assets/Scripts/Puzzle/SoundPuzzle.cs:3]: Unused using statement

File: Assets/Scripts/Puzzle/WaterPuzzle.cs [hunk 12]
```diff
@@ -0,0 +1,27 @@
+using System.Collections;
```
Comment (mbajaman) [inline Assets/Scripts/Puzzle/WaterPuzzle.cs]: Unused using statement

**Review by mbajaman — COMMENTED** [review]
Just minor formatting changes

File: Assets/Scripts/Puzzle/SoundPuzzle.cs (hunk 11 above)
Comment (jake-pauls) [reply Assets/Scripts/Puzzle/SoundPuzzle.cs:3]: This is required for IEnumerator since it isn't used as a generic in the context of a coroutine 👍

**Comment by jake-pauls** [comment]
Should be cleaned up now. Let me know! 👍

**Review by mbajaman — APPROVED** [review]
Looks gooood 🚀🚀

**Pull request merged**